website_sale_hide_empty_category/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_has_product_recurcive
```

Comment:
You missed this **recurcive** :wink:

Reply:
Hi @francoisdelarbre,
Could you rename your file `website_sale_hide_empty_category/tests/test_has_product_recurcive.py` to `website_sale_hide_empty_category/tests/test_has_product_recursive.py` and update the `__init__.py` file in consequence ?

After that, I think your PR will be accepted. :)